content/en/blog/falco-modern-bpf/index.md

```diff
@@ -0,0 +1,206 @@
+
+You understood well: No more missing drivers, and no more painful local builds requiring the much-loved **KERNEL HEADERS**.
+
+### BPF Ring Buffer map
+
+Today, whenever a BPF program needs to send data to userspace, it first copies it into a BPF map, and then it pushes its content to a shared buffer located between userspace and kernel: The so-called **`perf-buffer`**.
+
+This solution works great, but it has two major shortcomings that prove to be inconvenient in practice: Inefficient APIs and extra data copying.
+
+The **`ring buffer`** can be considered an evolution of the **`perf buffer`**. We still use a shared buffer but with some advantages.
+
+First, the APIs to interact with are more efficient. Second, and more importantly, we now have the possibility of writing data directly into this shared buffer, without having to write it twice!
+
+This is a game changer in scenarios where high throughput is required, like running Falco, since it could save many cycles during the collection phase.
+
+### BTF-enabled program
+
+Kernel engineers recently introduced a sort of debugging information for BPF programs/maps. This is called [BTF](https://docs.kernel.org/bpf/btf.html) and stands for "BPF Type Format". This feature shook the foundations of the BPF world because it finally offers the possibility to write code without BPF helpers like the famous `bpf_probe_read()`.
+
+This will not only increase the readability of the BPF code but will also reduce the bytecode dimension, allowing the crafting of smaller and more efficient programs!
+
+### BPF global variables
+
+The addition of global variables points in the same direction as BTF: Simplify the code experience, increase the readability and reduce the performance overhead due to BPF helpers.
+
+While BTF allows to deference kernel pointer without the use of the`bpf_probe_read()` helper, global variables allow to access BPF maps without the use of helpers like `bpf_map_lookup_elem()`.
+
+You can find further technical information about these 2 new features in the [probe proposal](https://github.com/falcosecurity/libs/blob/master/proposals/20220329-modern-bpf-probe.md#new-bpf-tracing-programs-kernel-version-55).
+
+### BPF skeleton
+
+Do you remember when we talked about Falco being a self-contained binary?
+
+Well the BPF skeleton concept allows us to achieve the dream:
+**Ship Falco as a unique, self-contained executable!**
+
+Under the hood, the probe is compiled once into Falco when the package is built. Hence, when you deploy Falco on different machines it will automatically inject the code without any extra effort on your side. Hard to believe? Try it [here](#modern-bpf-in-action).
+
+### Multi-arch support
+
+The modern BPF probe also supports multiple architectures by design. The actual targets are `x86_64`, `arm64`, and `s390x`, but new ones can be added at any time.
```

Comment:
Perhaps a "special thanks to Hendrick" here? Since he was the guy mainly involved in testing the probe on s390x!